### PR TITLE
Update Dockerfile to host openQA in Leap 42.2

### PR DESCRIPTION
**Pull request by wnereiz**

### CODE REVIEW

File: docker/openqa_data/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM opensuse:42.1
-MAINTAINER Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz(@github)
+FROM opensuse:42.2
+MAINTAINER Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz<wnereiz@eienteiland.org>
```
Comment (okurz) [inline docker/openqa_data/Dockerfile]: add space before "<"

**Comment by coolo** [comment]
btw: I assume RH is no longer interested in this - since they have fedora packages now

**Comment by wnereiz** [comment]
> btw: I assume RH is no longer interested in this - since they have fedora packages now
Yes, but I found docker is easy for deploying local openQA test case dev environment. So I suggest we keep it.

**Comment by okurz** [comment]
Sure, no doubt

**Comment by coolo** [comment]
please rebase to get rid of the 'merge' commit

**Comment by foursixnine** [comment]
> Yes, but I found docker is easy for deploying local openQA test case dev environment. So I suggest we keep it.

@wnereiz Would you mind updating the docs aswell?

**Comment by wnereiz** [comment]
> please rebase to get rid of the 'merge' commit
Rebased but checks failed

**Comment by wnereiz** [comment]
> @wnereiz Would you mind updating the docs aswell?

Which place need to be updated in the docs?

**Pull request closed without merging**